tests/pkg/tests/observability_alert_test.go

```diff
@@ -237,6 +247,83 @@ var _ = Describe("Observability:", func() {
 		klog.V(3).Infof("Successfully deleted CM: thanos-ruler-custom-rules")
 	})
 
+	It("[P2][Sev2][Observability][Stable] Should have alert named Watchdog forwarded to alertmanager generated (alert/g0)", func() {
```

Comment:
Suggest to use [Integration] firstly. we can move it to stable once it is stable.

Reply:
done.